test/acceptance/features/bindAppToMultipleServices.feature

```diff
@@ -6,12 +6,32 @@ Feature: Bind a single application to multiple services
     Background:
         Given Namespace [TEST_NAMESPACE] is used
         * Service Binding Operator is running
-        * PostgreSQL DB operator is installed
+        * OLM Operator "backend" is running
 
     Scenario: Bind two backend services by creating 2 SBRs to a single application
-        Given Imported Nodejs application "nodejs-app" is running
-        * DB "db-demo-1" is running
-        * DB "db-demo-2" is running
+        Given Generic test application "myapp-2sbrs" is running
```

Comment:
Focus of this PR (according to the summary) is about creating a test for binding two services with single SB. Hence, this scenario should not be touched here. Please revert the changes, and apply the in another focused PR.

Reply:
Changes reverted.